app/renderer/actions/Inspector.js

```diff
@@ -500,3 +505,98 @@ export function setActionArg (index, value) {
     dispatch({type: SET_ACTION_ARG, index, value});
   };
 }
+
+/**
+ * Ping server every 30 seconds to prevent `newCommandTimeout` from killing session
```

Comment:
We set `newCommandTimeout` to zero by default, so I'm wondering why this method is still needed. It would be logically to assume that if a user still sets `newCommandTimeout` then he expects a session to be terminated in case of inactivity

Reply:
I wondered about this too, my guess was that it's for cloud services that don't support the cap. I'd love to get rid of this logic though!

Reply:
@dpgraham can you remind us why we have this keepalive logic?

Reply:
@jlipps yeah that's because certain cloud providers shut down, regardless of if they set `newCommandTimeout`.